src/entities/user.entity.ts

```diff
@@ -17,4 +18,10 @@ export class User extends Base {
 
   @Column({ nullable: true })
   gender?: string;
+
+  @Column({ nullable: true })
+  image?: string; // Image as Base64
+
+  @Column({ nullable: false, default: Roles.User })
```

Comment:
Only a tiny comment, nullable by default is false, so it is not necessary to explicitly write it (: